datacenter/dtr/2.1/guides/high-availability/index.md

```diff
@@ -34,11 +34,15 @@ the DTR cluster.
 When sizing your DTR installation for high-availability,
 follow these rules of thumb:
 
-* Don't create a DTR cluster with just two replicas. Your cluster
-won't tolerate any failures, and it's possible that you experience performance
-degradation.
+* Don't create a DTR cluster with an even number replicas. Your cluster
+won't tolerate any failures if the cluster splits in half. This is
+very difficult to recover from.
 * When a replica fails, the number of failures tolerated by your cluster
 decreases. Don't leave that replica offline for long.
+* When you want to fix your DTR cluster, make certain that you remove any
+broken replicas before you try to add a new replica.  The DTR bootstrapper
```

Comment:
Minor nitpick, we don't use the word 'bootstrapper' in the docs. Either 'installer' or 'when deploying DTR...'.